components/infobox/wikis/halo/infobox_map_custom.lua

```diff
@@ -0,0 +1,81 @@
+---
+-- @Liquipedia
+-- wiki=halo
+-- page=Module:Infobox/Map
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Map = require('Module:Infobox/Map')
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local MapModes = require('Module:MapModes')
+
+local CustomMap = Class.new()
+
+local CustomInjector = Class.new(Injector)
+
+local _args
+local _game
+
+local _GAME = mw.loadData('Module:GameVersion')
+
+function CustomMap.run(frame)
+	local customMap = Map(frame)
+	customMap.createWidgetInjector = CustomMap.createWidgetInjector
+	customMap.getCategories = CustomMap.getCategories
+	_args = customMap.args
+	return customMap:createInfobox(frame)
+end
+
+function CustomMap:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:addCustomCells(widgets)
+	table.insert(widgets, Cell{
+		name = 'Location',
+		content = {_args.location}
+	})
+	table.insert(widgets, Cell{
+		name = 'Type',
+		content = {_args.type}
+	})
+	table.insert(widgets, Cell{
+		name = 'Game Version',
+		content = {CustomMap._getGameVersion()},
+		options = {makeLink = true}
+	})
+	table.insert(widgets, Cell{
+		name = 'Game Modes',
+		content = {CustomMap._getGameMode(_args.mode)}
```

Comment:
```suggestion
		content = {CustomMap._getGameMode()}
```
you do not use it in the other function (since you access it via _args.mode there too)